### PR TITLE
NO-ISSUE: Add multiarch OCP 4.12 release images

### PR DESCRIPTION
**Pull request by mkowalski**
/cc @osherdp

### CODE REVIEW

**Review by osherdp — CHANGES_REQUESTED** [review]
I think you'll need to do "make generate"

**Review by osherdp — APPROVED** [review]

File: data/default_release_images.json [hunk 1]
```diff
@@ -85,5 +85,17 @@
...
+      "cpu_architecture": "multi",
+      "cpu_architectures": [
+          "x86_64",
+          "arm64",
+          "ppc64le",
+          "s390x"
+      ],
+      "url": "quay.io/openshift-release-dev/ocp-release:4.12.0-rc.4-multi",
+      "version": "4.12.0-rc.4-multi"
+  }
```
Comment (osherdp) [inline data/default_release_images.json:100]: indentation is weird here. can you fix it?

**Review by osherdp — CHANGES_REQUESTED** [review]

File: data/default_release_images.json (hunk 1 above)
Comment (mkowalski) [reply data/default_release_images.json]: Yes you are right

**Review by osherdp — APPROVED** [review]

**Comment by mkowalski** [comment]
/retitle NO-ISSUE: Add multiarch OCP 4.12 release images

**Comment by mkowalski** [comment]
/close

Done via #4795

**Pull request closed without merging**